api/src/utilities/application-export-helpers.ts

```diff
@@ -22,11 +24,16 @@ export const getExportHeaders = (
   maxHouseholdMembers: number,
   multiSelectQuestions: MultiselectQuestion[],
   timeZone: string,
+  user: User,
```

Comment:
suggestion(blocking): If we are going to switch to using the user to get the feature flags we should keep it consistent for all feature flags. Specifically the "enableFullTimeStudentQuestions" is getting passed in but we are getting "enableAdaOtherOption" from the user jurisdictions.

I think it makes sense to not explicitly pass in each feature flag that we need so can you switch the "enableFullTimeStudentQuestion" to get it from the user jurisdictions here. That way too we can remove a db call in application-exporter.service that gets the jurisdictions.